src/plugins/media/schema.js

```diff
@@ -21,24 +37,36 @@ function normalizeMedia(change, error) {
     }
 }
 
-export default {
-    blocks: {
-        // TODO: do we actually want to keep multiple versions?
-        media: {
-            nodes: [
-                { match: [{ type: 'image' }], min: 1 },
-            ],
-            data: {
-                alt: Boolean,
+export default function schema({ inlines }) {
+    return {
+        blocks: {
+            // TODO: do we actually want to keep multiple versions?
+            media: {
+                nodes: [
+                    { match: { type: 'image' }, min: 1 },
+                    { match: { type: 'media_alt' }, min: 1, max: 1 },
+                ],
+                data: {
+                    alt: Boolean,
+                },
+                normalize: normalizeMedia,
             },
-            normalize: normalizeMedia,
-        },
-        // TODO: if we do, add target attribute.
-        image: {
-            isVoid: true,
-            data: {
-                src: Boolean,
+            media_alt: {
+                nodes: [{
+                    match: [
+                        ...inlines.map(type => ({ type })),
+                        { object: 'text' },
+                    ],
+                }],
+                marks: [],
```

Comment:
There's no need to specify an empty `marks`.

Reply:
`marks: undefined` will allow every mark